<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerGroupDialogUI</class>
 <widget class="QWidget" name="ServerGroupDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="m_networkLabel">
     <property name="text">
      <string>Network name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_identityLabel">
     <property name="text">
      <string>Identity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_identityCBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="m_identityCBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>Choose an existing Identity or click the Edit button to add a new Identity or edit an existing one. The Identity will identify you and determine your nickname when you connect to the network.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="m_editIdentityButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Edit...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_commandsLabel">
     <property name="text">
      <string>Commands:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_commandEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="KLineEdit" name="m_commandEdit">
     <property name="whatsThis">
      <string>&lt;qt&gt;Optional. This command will be sent to the server after connecting. Example: &lt;b&gt;/msg NickServ IDENTIFY &lt;i&gt;konvirocks&lt;/i&gt;&lt;/b&gt;. This example is for the Libera network, which requires users to register their nickname with a password and login when connecting. &lt;i&gt;konvirocks&lt;/i&gt; is the password for the nickname given in Identity. You may enter more than one command by separating them with semicolons.&lt;/qt&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KLineEdit" name="m_nameEdit">
     <property name="whatsThis">
      <string>Enter the name of the Network here. You may create as many entries in the Server List screen with the same Network as you like.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Servers:</string>
     </property>
     <property name="buddy">
      <cstring>m_serverLBox</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Auto Join Channels:</string>
     </property>
     <property name="buddy">
      <cstring>m_channelLBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="m_autoConnectCBox">
     <property name="whatsThis">
      <string>Check here if you want Konversation to automatically connect to this network whenever you open Konversation.</string>
     </property>
     <property name="text">
      <string>Co&amp;nnect on application start up</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QWidget" name="serverWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="6">
       <widget class="QListWidget" name="m_serverLBox">
        <property name="whatsThis">
         <string>This is a list of IRC Servers in the network. When connecting to the network, Konversation will attempt to connect to the top server first. If this fails, it will attempt the second server. If this fails, it will attempt the third, and so on. At least one server must be specified. Click a server to highlight it.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="m_addServerButton">
        <property name="text">
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="m_changeServerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>E&amp;dit...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="m_removeServerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="serverSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="m_upServerBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="m_downServerBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QWidget" name="channelWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="6">
       <widget class="QListWidget" name="m_channelLBox">
        <property name="whatsThis">
         <string>Optional. This is a list of the channels that will be automatically joined once Konversation has connected to a server. You may leave this blank if you wish to not automatically join any channels.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="m_addChannelButton">
        <property name="text">
         <string>Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="m_changeChannelButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ed&amp;it...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="m_removeChannelButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="channelSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>255</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="m_upChannelBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="m_downChannelBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_nameEdit</tabstop>
  <tabstop>m_identityCBox</tabstop>
  <tabstop>m_editIdentityButton</tabstop>
  <tabstop>m_commandEdit</tabstop>
  <tabstop>m_autoConnectCBox</tabstop>
  <tabstop>m_serverLBox</tabstop>
  <tabstop>m_addServerButton</tabstop>
  <tabstop>m_changeServerButton</tabstop>
  <tabstop>m_removeServerButton</tabstop>
  <tabstop>m_upServerBtn</tabstop>
  <tabstop>m_downServerBtn</tabstop>
  <tabstop>m_channelLBox</tabstop>
  <tabstop>m_addChannelButton</tabstop>
  <tabstop>m_changeChannelButton</tabstop>
  <tabstop>m_removeChannelButton</tabstop>
  <tabstop>m_upChannelBtn</tabstop>
  <tabstop>m_downChannelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
